zetaclient/keys/keys.go

```diff
@@ -12,23 +12,25 @@ import (
 	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
 	ckeys "github.com/cosmos/cosmos-sdk/crypto/keyring"
 	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
-	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkTypes "github.com/cosmos/cosmos-sdk/types"
```

Comment:
```suggestion
	sdk "github.com/cosmos/cosmos-sdk/types"
```
We should not use uppercase in import